pkg/handler/v2/etcdbackupconfig/etcdbackupconfig_test.go

```diff
@@ -147,21 +148,21 @@ func TestGetEndpoint(t *testing.T) {
 	}{
 		{
 			Name:               "get etcdbackupconfig that belongs to the given cluster",
-			EtcdBackupConfigID: "id-1",
+			EtcdBackupConfigID: etcdbackupconfig.GenEtcdBackupConfigID("test-1", test.GenDefaultCluster().Name),
```

Comment:
Can you also check situation when cluster name contains multiple `-` like `test-abc` and `test-ab-cd`